zebra-chain/src/transaction/auth_digest.rs

```diff
@@ -45,6 +45,17 @@ impl From<&Transaction> for AuthDigest {
     }
 }
 
+impl From<Arc<Transaction>> for AuthDigest {
+    /// Computes the authorizing data commitment for a transaction.
+    ///
+    /// # Panics
+    ///
+    /// If passed a pre-v5 transaction.
+    fn from(transaction: Arc<Transaction>) -> Self {
+        transaction.as_ref().into()
+    }
+}
```

Comment:
👍

Reply:
It seems a little odd that a `From` impl should be sometimes panicking.